### PR TITLE
Modify checked and unchecked box icons

### PR DESCRIPTION
**Pull request by n0toose**
- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

### CODE REVIEW

File: images/Checked.xpm [hunk 1]
```diff
@@ -1,88 +1,82 @@
 /* XPM */
-static const char * const checked_xpm[] = {
-"15 15 70 1",
+static char * Checked_xpm[] = {
```
Comment (purplesyringa) [inline images/Checked.xpm]: Maybe make it `const char* const` still?

Comment (n0toose) [reply images/Checked.xpm]: TO-DO: Never use GIMP again.

Comment (nbsp) [reply images/Checked.xpm]: stale

File: images/Checked.xpm [hunk 2]
```diff
@@ -1,88 +1,82 @@
 /* XPM */
-static const char * const checked_xpm[] = {
-"15 15 70 1",
+static const char * const Checked_xpm[] = {
```
Comment (purplesyringa) [inline images/Checked.xpm]: Interesting. You changed the variable name but it still builds. Why does this even work?

Comment (n0toose) [reply images/Checked.xpm]: I don't know, but it shouldn't work.

**Review by Semisol — APPROVED** [review]

**Review by nbsp — APPROVED** [review]

**Pull request merged**